docker/install/ubuntu_install_sphinx.sh

```diff
@@ -20,14 +20,14 @@ set -e
 set -u
 set -o pipefail
 
-# NOTE: install docutils < 0.17 to work around https://github.com/readthedocs/sphinx_rtd_theme/issues/1115
+# pin sphinx<7 because https://github.com/readthedocs/sphinx_rtd_theme/issues/1465
```

Comment:
Sphinx 7 should work with https://github.com/tlc-pack/tlcpack-sphinx-addon/pull/11

Reply:
installing https://github.com/tlc-pack/tlcpack-sphinx-addon solved the problem. Thanks a lot!